src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
 export interface Figure {
-
+  shape: string,
```

Comment:
Create an [enum](https://www.typescriptlang.org/docs/handbook/enums.html) `Color`, so you don't have to mess with bare strings.